.gitignore

```diff
@@ -12,8 +12,7 @@ _test
 .ruby-version
 .vagrant
 .vscode/*
-!.vscode/launch.json
-!.vscode/tasks.json
+!.vscode/launch.json !.vscode/tasks.json
```

Comment:
I think these should be on two separate lines still

Reply:
@markphelps Idk what happened here, will change this up.